### ISSUE TITLE
Implement a conv_cond_concat function

### ISSUE THREAD

**Issue by metatl** [keras-team/keras#12766]
In tensorflow, we can do
def conv_cond_concat(x, y):
  """Concatenate conditioning vector on feature map axis."""
  x_shapes = x.get_shape()
  y_shapes = y.get_shape()
  return concat([
    x, y*tf.ones([x_shapes[0], x_shapes[1], x_shapes[2], y_shapes[3]])], 3)

In keras functional api, I don't know how to do this with two input layers that have different sizes. Anyone knows how to achieve this in keras 2.0.5?

**Comment by jvishnuvardhan** [comment]
Please fill the issue [template](https://github.com/keras-team/keras/issues/new?template=a--tensorflow-backend-users.md). Could you update them if they are relevant in your case, or leave them as N/A? Along with the template, please provide as many details as possible to find the root cause of the issue. It would be great if you can provide a small code to reproduce the error. Thanks!

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will open a new issue. Thanks!
